### PR TITLE
Logger added

### PR DESCRIPTION
**Pull request by hpavlo**

### CODE REVIEW

File: src/main/java/mate/academy/service/AuthenticationServiceImpl.java [hunk 1]
```diff
@@ -2,11 +2,15 @@
...
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Method login was called. Params: login='{}'", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
             throw new AuthenticationException("Username or password are incorrect");
```
Comment (YuriiHryniv) [inline src/main/java/mate/academy/service/AuthenticationServiceImpl.java:16]: Personally, I would make this exception message a constant.

File: src/main/java/mate/academy/Main.java [hunk 2]
```diff
@@ -6,15 +6,19 @@
...
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
             user = authenticationService.login("bob", "1234");
```
Comment (YuriiHryniv) [inline src/main/java/mate/academy/Main.java:19]: I would make these fields constats too etc.

**Review by YuriiHryniv — APPROVED** [review]
Good!

File: src/main/java/mate/academy/Main.java [hunk 3]
```diff
@@ -6,15 +6,21 @@
...
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
-            user = authenticationService.login("bob", "1234");
+            user = authenticationService.login(LOGIN, PASSWORD);
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Can't login", e);
```
Comment (KostyaNikiforov) [inline src/main/java/mate/academy/Main.java:23]: Add info about the user to the message

**Review by KostyaNikiforov — CHANGES_REQUESTED** [review]

**Review by KostyaNikiforov — APPROVED** [review]

**Pull request closed without merging**